vllm/spec_decode/spec_decode_worker.py

```diff
@@ -287,7 +287,10 @@ def _configure_model_sampler_for_spec_decode(self):
         """
         (self.scorer_worker.model_runner.model.sampler.include_gpu_probs_tensor
          ) = True
+        (self.scorer_worker.model_runner.model.sampler.
+         should_modify_greedy_probs_inplace) = True
```

Comment:
Can we put this behind an interface so refactors to the worker / model / sampler do spread everywhere?

Reply:
Not sure if there is a clean way to do this, as the line above sets `include_gpu_probs_tensor` before this PR also change the decoupled (this PR) `should_modify_greedy_probs_inplace`. Seems the scorer worker is passed around as `WorkerBase` so would need to add change that class which is not ideal. Am I missing something?

Reply:
Oh right.. sorry I must have misread.